### ISSUE TITLE
'private_action' and 'yoda_condition' are not a valid rule identifier

### ISSUE THREAD

**Issue by hani-ibrahim** [realm/SwiftLint#1997]
Updating to the latest version 0.24.0 and running `swiftlint` in terminal gives this two errors
```
Loading configuration from '.swiftlint.yml'
configuration error: 'private_action' is not a valid rule identifier
configuration error: 'yoda_condition' is not a valid rule identifier
Valid rule identifiers:
...
```

**Comment by ornithocoder** [comment]
Hi @hani-ibrahim, how did you upgrade to 0.24.0? Was it using CocoaPods or brew? I'm asking this because one of my coworkers had a similar issue a long time ago. He had two swiftlints on his computer, one he installed using brew and the one on the project (via CocoaPods). When running via terminal, he was using the older one installed via brew (which didn't include the new rules we were using).

**Comment by marcelofabri** [comment]
Those rules are not available on 0.24.0, only on `master`.

**Comment by hani-ibrahim** [comment]
@ornithocoder I’m using brew
@marcelofabri what about releasing a new version? 🤔
